tests/previewjs.ts

```diff
@@ -6,7 +6,8 @@ export async function test(options: RunOptions) {
 		...options,
 		repo: 'fwouts/previewjs',
 		branch: 'main',
-		build: 'turbo build',
-		test: ['turbo e2e-test -- --workers=1']
+		build: 'vite-ecosystem-ci:build',
+		beforeTest: 'pnpm vite-ecosystem-ci:test',
```

Comment:
beforeTest can be used for additional setup, using it to run tests twice is very unusual. could it be removed or simplified?

Reply:
This was a typo! I'm not sure why you're not seeing the latest version, maybe a GitHub bug?

Reply:
It looks fine for me:
```
		build: 'vite-ecosystem-ci:build',
		beforeTest: 'pnpm vite-ecosystem-ci:before-test',
		test: ['vite-ecosystem-ci:test']
```

Reply:
not sure what happened, maybe a glitch in gh mobile ui. Command looks fine now, do you have to prefix it with pnpm though? build, beforeTest and test use 3 different styles for commands (script name, shell command, array with script name). If all of them are single scripts in package.json, would it hurt to use simple script name for all?

Reply:
@dominikg you're right, for some reason I was under the impression that the `beforeTest` option had to be a shell command, but that's not the case. Fixed!

Reply:
thanks